packages/smooth_app/lib/helpers/analytics_helper.dart

```diff
@@ -79,7 +79,15 @@ class AnalyticsHelper {
     return event;
   }
 
-  static void initMatomo(final BuildContext context) {
+  static void initMatomo(
+    final BuildContext context,
+    final bool screenshotMode,
+  ) {
+    if (screenshotMode) {
+      setCrashReports(false);
```

Comment:
What about putting the same logic in initSentry

Reply:
It's not that I particulary care about Matomo rather than Sentry; it's just that `initMatomo` froze the app in screenshot mode. And I'm sort of forced to call `initMatomo`so that the next matomo calls don't crash.
In screenshot mode I don't even call `initSentry`.